packages/maker.js/src/core/svg.ts

```diff
@@ -412,7 +413,8 @@ namespace MakerJs.exporter {
             fillRule: "evenodd",
             fontSize: '9pt',
             useSvgPathOnly: true,
-            viewBox: true
+            viewBox: true,
+            className:""
```

Comment:
I think this is not needed, as line 420 will merge it if it exists.

Reply:
Removed `className `from here